vulyk/utils.py

```diff
@@ -86,3 +87,11 @@ def get_tb():
     :return: traceback of the most recent exception
     """
     return sys.exc_info()[2]
+
+
+def get_template_path(app, name):
+    for x in iter(app.jinja_loader.list_templates()):
```

Comment:
Yes, it is my mistake. In this situation need not `iter` 
Hmm, if `app.config['TEMPLATE_BASE_FOLDERS']` is None, error.
I will add conditions for this situation